server/neptune/gateway/event/pull_request_review_handler.go

```diff
@@ -55,18 +58,19 @@ func (p *PullRequestReviewWorkerProxy) handle(ctx context.Context, event PullReq
 	}
 
 	// Ignore non-approval events
-	if event.Action != Approved {
+	if event.State != Approved {
+		p.Logger.InfoContext(ctx, fmt.Sprintf("not an approval event: %s", event.State))
```

Comment:
do we need this `Info` log here? This might end up spamming our logs since all the non-approval PR events would write this log

Reply:
it's only for initial testing